webpack/scenes/Subscriptions/Manifest/ToggleManifestSCA.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch } from '@patternfly/react-core';
+
+class SimpleSwitch extends React.Component {
```

Comment:
A better name for this component (and its file) would be `SimpleContentAccessToggle`. We don't need to use 'manifest' since we are already in a folder called /Manifest/